rustfmt-core/rustfmt-lib/tests/target/issue-3068.rs

```diff
@@ -0,0 +1,15 @@
+// rustfmt-hard_tabs: true
+
+macro_rules! foo {
+	($bar: expr, $t: ty) => {
+		$bar(|x| {
+			if x {
+				None;
```

Comment:
Let's remove the extra statement so this matches the snippet provided in the issue
```suggestion
```

Reply:
Done. Keeping the semicolon so that we force the block; otherwise, the macro will wrap into one line.

Reply:
Could we use `single_line_if_else_max_width` instead? Want to keep it as close to the original issue as possible
```
// rustfmt-single_line_if_else_max_width: 10
```

Reply:
yep, fixed

Reply:
thanks! will merge once CI jobs finish